dev-haskell/hjsmin/hjsmin-0.2.1.ebuild

```diff
@@ -17,19 +15,12 @@ LICENSE="BSD"
 SLOT="0/${PV}"
 KEYWORDS="~amd64"
 
-PATCHES=(
-	"${FILESDIR}/${PN}-0.2.0.4-fix-tests.patch"
-)
-
-RDEPEND="
-	>=dev-haskell/language-javascript-0.6:=[profile?] <dev-haskell/language-javascript-0.8:=[profile?]
+RDEPEND=">=dev-haskell/language-javascript-0.6:=[profile?] <dev-haskell/language-javascript-0.8:=[profile?]
 	>=dev-haskell/optparse-applicative-0.7:=[profile?]
-	>=dev-haskell/text-1.2:=[profile?] <dev-haskell/text-1.3:=[profile?]
-	>=dev-lang/ghc-8.8.1:=
+	>=dev-haskell/text-2:=[profile?]
```

Comment:
`dev-haskell/text` is one of those packages that comes bundled with ghc, so we are tied to the version of ghc on the system. In the case of `ghc-9.2.8` this is `dev-haskell/text-1.2.5.0`. As of the 9.4 series (which requires some significant porting work) `text` reaches version 2 as part of ghc. When we get there, we can use ebuilds such as this one which requires `>=dev-haskell/text-2`.

Reply:
> `dev-haskell/text` is one of those packages that comes bundled with ghc, so we are tied to the version of ghc on the system.

@solpeth, actually I think we _can_ upgrade `text` independent of what comes with `ghc` (see #1386), but it would be a lot of work to make sure everything is working with `>=text-2`, hence why it is unkeyworded.

Reply:
It doesn't look like they changed the code at all, only changed the version range for `text`, so it will likely work with `text-1.2.5.0`.

https://github.com/erikd/hjsmin/pull/39/files

Reply:
@hololeap you are right - I forgot that they removed their dependency on text in libghc upstream. Thanks for looking into the text lower bound too! If Miezhiko wants to they can adjust it, otherwise I will endeavour to get around to it over the weekend.

Reply:
I'll be able to look into that only on next weekend, but if you will do something earlier = good